<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>346</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>346</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>346</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Калькулятор</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(128, 151, 114);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="result">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="styleSheet">
           <string notr="true">font: 700 9pt &quot;Comic Sans MS&quot;;
color: rgb(35, 41, 29);
background-color: rgb(158, 187, 133);</string>
          </property>
          <item>
           <property name="text">
            <string>Обычный</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Кредитный</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Депозитный</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_change">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 700 9pt &quot;Comic Sans MS&quot;;
color: rgb(35, 41, 29);
background-color: rgb(163, 192, 137);</string>
          </property>
          <property name="text">
           <string>Сменить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_x">
          <property name="styleSheet">
           <string>font: 700 9pt &quot;Comic Sans MS&quot;;
color: rgb(35, 41, 29);</string>
          </property>
          <property name="text">
           <string>X = </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="styleSheet">
           <string notr="true">font: 700 9pt &quot;Comic Sans MS&quot;;
color: rgb(35, 41, 29);
background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_makegraphic">
          <property name="styleSheet">
           <string notr="true">font: 700 9pt &quot;Comic Sans MS&quot;;
color: rgb(35, 41, 29);
background-color: rgb(163, 192, 137);</string>
          </property>
          <property name="text">
           <string>Бахнуть график</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font: 700 9pt &quot;Comic Sans MS&quot;;</string>
          </property>
          <property name="text">
           <string>min X =</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_min_x">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>-10</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">font: 700 9pt &quot;Comic Sans MS&quot;;</string>
          </property>
          <property name="text">
           <string>min Y =</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_min_y">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>-10</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">font: 700 9pt &quot;Comic Sans MS&quot;;</string>
          </property>
          <property name="text">
           <string>max X =</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_max_x">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>10</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">font: 700 9pt &quot;Comic Sans MS&quot;;</string>
          </property>
          <property name="text">
           <string>max Y =</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_max_y">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>10</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_one">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_two">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_nine">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="pushButton_dot">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="pushButton_tan">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>tan</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="pushButton_asin">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>asin</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_eight">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton_three">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QPushButton" name="pushButton_atan">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>atan</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="pushButton_log">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>log</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_mod">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>%</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_leftbracket">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>(</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QPushButton" name="pushButton_ln">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>ln</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="pushButton_acos">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>acos</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_four">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="pushButton_equals">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(147, 210, 95);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="pushButton_cos">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>cos</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="pushButton_sin">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>sin</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton_zero">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_five">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="pushButton_sqrt">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>sqrt</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_seven">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_rightbracket">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_six">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButton_clear">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QPushButton" name="pushButton_pow">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>^</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QPushButton" name="pushButton_plus">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="pushButton_minus">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="pushButton_multiply">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="pushButton_divide">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="pushButton_x">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;</string>
          </property>
          <property name="text">
           <string>х</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="pushButton_remove">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(177, 211, 150);
font: 700 9pt &quot;Comic Sans MS&quot;;
</string>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QCustomPlot" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
